app/src/test/java/org/simple/clinic/patient/fuzzy/PercentageFuzzerTest.kt

```diff
@@ -3,44 +3,50 @@ package org.simple.clinic.patient.fuzzy
 import com.google.common.truth.Truth.assertThat
 import junitparams.JUnitParamsRunner
 import junitparams.Parameters
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
+import org.simple.clinic.util.TestClock
+import org.threeten.bp.LocalDate
 
 @RunWith(JUnitParamsRunner::class)
 class PercentageFuzzerTest {
 
-  @Parameters(value = [
-    "0|0.1|0|0",
-    "5|0.1|4|5",
-    "5|0.5|2|7",
-    "5|0.8|1|9",
-    "5|1.0|0|10",
-    "25|0.1|22|27",
-    "25|0.5|12|37",
-    "25|0.8|5|45",
-    "25|1.0|0|50",
-    "43|0.1|38|47",
-    "43|0.5|21|64",
-    "43|0.8|8|77",
-    "43|1.0|0|86"
-  ])
+  private val clock = TestClock()
+
+  @Before
+  fun setUp() {
+    clock.setYear(2000)
```

Comment:
We probably should. I'll add this task to pivotal.

Technically, `clock.resetToEpoch()` is not necessary because `setYear` does not take the current year into account.

Reply:
That method was meant to be used in cases where you need to reset the clock inside a test case.

Reply:
Ah, I completely forgot that this is not an Android test. Modifications to `TestClock` are limited to the scope of this test.